sitemap.svg

```diff
@@ -0,0 +1,11 @@
+PlantUML version 1.2020.06(Sun Apr 05 05:38:32 PDT 2020)
+(APACHE source distribution)
+Java Runtime: Java(TM) SE Runtime Environment
+JVM: Java HotSpot(TM) Client VM
+Java Version: 1.8.0_221-b11
+Operating System: Windows 10
+Default Encoding: Cp1252
+Language: en
+Country: US
+--></g></svg>
```

Comment:
are these auto--generated? if so then its, fine
But if not could we get a newline for eof?